src/components/molecules/OcTable/OcTable.vue

```diff
@@ -1,82 +1,228 @@
+          v-for="(item, trIndex) in tableData"
+          :key="`oc-tbody-tr-${item[idKey] || trIndex}`"
+          :ref="`row-${trIndex}`"
+          v-bind="extractTbodyTrProps(item, trIndex)"
+          :data-file-id="item.id"
+          :draggable="dragDrop"
+          @click.native="$emit(constants.EVENT_TROW_CLICKED, item)"
+          @contextmenu.native="
+            $emit(constants.EVENT_TROW_CONTEXTMENU, $refs[`row-${trIndex}`][0], $event, item)
+          "
+          @hook:mounted="$emit(constants.EVENT_TROW_MOUNTED, item, $refs[`row-${trIndex}`][0])"
+          @dragstart.native.stop="dragStart(item)"
+          @drop.native.stop="dropRowEvent(item.id, $event)"
+          @dragenter.native.prevent.stop="dropRowStyling(item.id, false, $event)"
+          @dragleave.native.prevent.stop="dropRowStyling(item.id, true, $event)"
+          @mouseleave="dropRowStyling(item.id, true, $event)"
+          @dragover.native.prevent.stop
         >
-          <span
-            v-if="field.headerType === 'slot'"
-            :key="field.name + 'Header'"
-            class="oc-table-thead-content"
+          <oc-td
+            v-for="(field, tdIndex) in fields"
+            :key="'oc-tbody-td-' + cellKey(field, tdIndex, item)"
+            v-bind="extractTdProps(field, tdIndex)"
           >
-            <slot :name="field.name + 'Header'" />
-          </span>
-          <!-- TODO: Vue warning "v-if/else branches must use unique keys." -->
-          <span
-            v-else
-            :key="field.name + 'Header'"
-            class="oc-table-thead-content"
-            v-text="extractFieldTitle(field)"
-          />
-          <oc-button
-            v-if="field.sortable"
-            :aria-label="getSortLabel(field.name)"
-            class="oc-button-sort"
-            variant="passive"
-            appearance="raw"
-            @click.stop="$emit(constants.EVENT_THEAD_CLICKED, field)"
-          />
-        </oc-th>
-      </oc-tr>
-    </oc-thead>
-    <oc-tbody>
-      <oc-tr
-        v-for="(item, trIndex) in tableData"
-        :key="`oc-tbody-tr-${item[idKey] || trIndex}`"
-        :ref="`row-${trIndex}`"
-        v-bind="extractTbodyTrProps(item, trIndex)"
-        :data-file-id="item.id"
-        :draggable="dragDrop"
-        @click.native="$emit(constants.EVENT_TROW_CLICKED, item)"
-        @contextmenu.native="
-          $emit(constants.EVENT_TROW_CONTEXTMENU, $refs[`row-${trIndex}`][0], $event, item)
+            <slot v-if="isFieldTypeSlot(field)" :name="field.name" :item="item" />
+            <template v-else-if="isFieldTypeCallback(field)">
+              {{ field.callback(item[field.name]) }}
+            </template>
+            <template v-else>
+              {{ item[field.name] }}
+            </template>
+          </oc-td>
+        </oc-tr>
+      </oc-tbody>
+
+      <!-- Collapsibles -->
+
+      <tbody
+        v-for="(item, index) in groupedItems"
+        v-else-if="groupingAllowed && selected !== 'None'"
```

Comment:
It's considered bad practice to have `v-for` and `v-(else-)if` on the same element. Only acceptable situation is when you want to skip certain elements of the `v-for`. In this case You'd execute the `v-for` with all the grouped items, just to skip it because of the `v-else-if` if grouping is not allowed.

Reply:
I tried to put `<template v-else-if...>` wrapper around but for some reason the code after the template gets inactive